src/less/modules/breadcrumbs.less

```diff
@@ -1,7 +1,9 @@
 .breadcrumb {
-  margin-left: 15px;
-  margin-top: -20px;
+  //TODO move breadcrumbs out into app-wide directive under navbar not within ui-view
```

Comment:
But perhaps it could be more extensively used. E.g., subviews for the tabs instead of separate directives. That would be awesome :heart_eyes: